src/test/java/org/apache/cassandra/sidecar/routes/StreamSSTableComponentHandlerTest.java

```diff
@@ -221,7 +221,8 @@ void testAccessDeniedToCertainComponents(VertxTestContext context)
     void failsWhenTableNameContainsInvalidCharacters(VertxTestContext context)
     {
         WebClient client = WebClient.create(vertx);
-        String testRoute = "/keyspaces/" + TEST_KEYSPACE + "/tables/i_❤_u/snapshots/snap/components/component-Data.db";
```

Comment:
The length of the line looks like 120 characters in IDE, but Checkstyle is complaining that it is having 121 characters due to the Unicode char. Added new line in between to avoid Checkstyle error.